android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -837,6 +839,45 @@ sealed class ContentAction : BrowserAction() {
     ) : ContentAction()
 }
 
+/**
+ * [BrowserAction] implementations related to translating a web content page.
+ */
+sealed class TranslationsAction : BrowserAction() {
+    /**
+     * Used to translate the page for a given [tabId].
+     */
+    data class TranslateAction(
+        override val tabId: String,
```

Comment:
Could we get the @property for everything in this class?

Reply:
Updated!